source/reference/limitations.txt

```diff
@@ -75,6 +72,7 @@ Unsupported Collection Types
 ----------------------------
 
 - Time-series collections are not supported.
+- Capped collections are currently not supported.
```

Comment:
[suggestion]

Would it make sense for all these lines to read similarly? If so: 

"Capped collections are not supported."

Reply:
Fixed